Kickstarter-iOS/Views/RewardCardContainerView.swift

```diff
@@ -158,6 +168,30 @@ public final class RewardCardContainerView: UIView {
     )
   }
 
+  private func configureBaseGradientView() {
+    _ = self.baseGradientView
+      |> roundedStyle(cornerRadius: Styles.grid(3))
+      |> \.layer.borderColor .~ UIColor.white.cgColor
+      |> \.layer.borderWidth .~ 2
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.clear
```

Comment:
Do we need the clear background on this?

Reply:
Seems not!